mojaloop-simulator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: "Helm Chart for the Mojaloop (SDK-based) Simulator"
 name: mojaloop-simulator
-version: 14.1.0
-appVersion: "sdk-scheme-adapter: v21.4.0; mojaloop-simulator: v13.0.1; thirdparty-sdk: v15.1.0"
+version: 15.0.0
```

Comment:
Not sure if chart versions should be left for a final commit to avoid mistakenly double bumping version. 

Bumping to new major because of sdk version jump.